### ISSUE TITLE
bug: wrong class name (Document became Documant)

### ISSUE THREAD

**Issue by dexcell** [zhangruiyu/FlutterJsonBeanFactory#43]
Flutter json bean factory wrong rename

How to reproduce:

class name:
DocumentRuleResponse

json
````
{
  "document_rules": [
    {
      "business_source_name": null,
      "vehicle_type_name": null,
      "loss_type": null,
      "loss_extension": null,
      "document_names": [
        "A",
        "B",
        "C"
      ]
    },
    {
      "business_source_name": null,
      "vehicle_type_name": "Car",
      "loss_type": "Partial Loss",
      "loss_extension": null,
      "document_names": [
        "D",
        "E"
      ]
    }
  ]
}
````

Result:
````
class DocumantRuleResponseDocumantRules with JsonConvert<DocumantRuleResponseDocumantRules> {
````

while  the correct one should be `Document` not `Documant`

<img width="273" alt="Screen Shot 2020-03-24 at 20 23 13" src="https://user-images.githubusercontent.com/41800/77430174-594bee00-6e0d-11ea-9959-9f1c038a7baf.png">

**Comment by SerggioC** [comment]
this is very weird. 
Why hasn't the author replied yet?

**Comment by zhangruiyu** [comment]
fixed in 4.0.6 and support null safety. https://github.com/zhangruiyu/FlutterJsonBeanFactory/releases/tag/4.0.6
